packages/button/src/Button/components/GenericButton.tsx

```diff
@@ -47,6 +47,7 @@ export type GenericProps = {
   onBlur?: (e: FocusEvent<HTMLElement>) => void
   component?: ComponentType<CustomButtonProps>
   classNameOverride?: string
+  "aria-describedby"?: string
```

Comment:
@dougmacknz I decided to add this to the button props so a consumer (or in this case consuming components like the Tooltip) doesn't need to use the custom component rendered to be able to pass in aria-described to the actual button (instead of the span that wraps it)